_content/canon/connection-with-pevious-deeds_anandajoti.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Pubbakammapilotika-Buddhāpadānaṁ: The Traditions about the Buddha (known as)
+The Connection with Previous Deeds, or Why the Buddha Suffered"
+authors:
+  - anandajoti
+external_url: "https://ancient-buddhist-texts.net/Texts-and-Translations/Connection-with-Previous-Deeds/Connection-with-Previous-Deeds.pdf"
```

Comment:
Homepage, not directly to the pdf please: https://ancient-buddhist-texts.net/Texts-and-Translations/Connection-with-Previous-Deeds/